dev-cpp/benchmark/benchmark-1.8.5.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_12 )
+
+inherit cmake-multilib python-single-r1
+
+DESCRIPTION="A microbenchmark support library"
+HOMEPAGE="https://github.com/google/benchmark/"
+SRC_URI="https://github.com/google/benchmark/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="0/$(ver_cut 1)"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="default-libcxx doc libpfm lto +exceptions test +tools"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	default-libcxx? ( sys-libs/libcxx[${MULTILIB_USEDEP}] )
+	libpfm? ( dev-libs/libpfm:= )
+"
+
+BDEPEND="
+	>=dev-build/cmake-3.10
+	test? ( dev-cpp/gtest[${MULTILIB_USEDEP}] )
+"
+
+RDEPEND="
+	tools? (
+		$(python_gen_cond_dep '
+			dev-python/numpy[${PYTHON_USEDEP}]
+			>=dev-python/scipy-1.10.0[${PYTHON_USEDEP}]
+		')
+
+		${PYTHON_DEPS}
+	)
+"
+
+PATCHES=( "${FILESDIR}/${P}-fix-documentation-installation.patch" )
+
+pkg_setup() {
+	use tools && python-single-r1_pkg_setup
+}
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DBENCHMARK_ENABLE_DOXYGEN="$(usex doc)"
+		-DBENCHMARK_ENABLE_EXCEPTIONS="$(usex exceptions)"
+		-DBENCHMARK_ENABLE_GTEST_TESTS="$(usex test)"
+		-DBENCHMARK_ENABLE_LTO="$(usex lto)"
```

Comment:
Could you check what exactly `+lto` does in the build system? If it **just** adds `-flto` to your `*FLAGS`, then we can get rid of the use flag in here.

Reply:
It seems that `BENCHMARK_ENABLE_LTO` do not only add `-flto` ([here](https://github.com/google/benchmark/blob/c19cfee61e136effb05a7fc8a037b0db3b13bd4c/CMakeLists.txt#L163-L181) and [here](https://github.com/google/benchmark/blob/c19cfee61e136effb05a7fc8a037b0db3b13bd4c/CMakeLists.txt#L249-L264)).